xml/System/DateTime.xml

```diff
@@ -46,351 +46,353 @@
+[!code-vb[System.DateTime.Instantiation#4](~/samples/snippets/visualbasic/System.DateTime/Instantiation.vb#4)]  
+[!code-csharp-interactive[System.DateTime.Instantiation#4](~/samples/snippets/csharp/System.DateTime/Instantiation.cs#4)]
+
+The <xref:System.DateTime.TryParse%2A> and <xref:System.DateTime.TryParseExact%2A> methods indicate whether a string is a valid representation of a <xref:System.DateTime> value in addition to performing the conversion.  
+
+#### Language-specific syntax for Visual Basic
+
+The following Visual Basic statement initializes a new <xref:System.DateTime> value.  
   
- Local time is relative to a particular time zone. A time zone is associated with a time zone offset, which is the displacement of the time zone measured in hours from the UTC origin point. In addition, local time is optionally affected by daylight saving time, which adds or subtracts an hour from the length of a day. Consequently, local time is calculated by adding the time zone offset to UTC and adjusting for daylight saving time if necessary. The time zone offset at the UTC origin point is zero.  
+[!code-vb[System.DateTime.Instantiation#2](~/samples/snippets/visualbasic/System.DateTime/Instantiation.vb#2)]  
+
+### DateTime values and their string representations  
+
+Internally, all <xref:System.DateTime> values are represented as the number of ticks (the number of 100-nanosecond intervals) that have elapsed since 12:00:00 midnight, January 1, 0001. The actual <xref:System.DateTime> value is independent of the way in which that value appears when displayed. The appearance of a <xref:System.DateTime> value is the string result of a formatting operation that converts a value to its string representation.  
   
- UTC time is suitable for calculations, comparisons, and storing dates and time in files. Local time is appropriate for display in user interfaces of desktop applications. Time zone-aware applications (such as many Web applications) also need to work with a number of other time zones.  
+The appearance of date and time values is dependent on culture, international standards, application requirements, and personal preference. The <xref:System.DateTime> structure offers flexibility in formatting date and time values through overloads of  <xref:System.DateTime.ToString%2A>. The default <xref:System.DateTime.ToString?displayProperty=nameWithType> method returns the string representation of a date and time value using the current culture's short date and long time pattern. The following example uses the default <xref:System.DateTime.ToString?displayProperty=nameWithType> method. It displays the date and time using the short date and long time pattern for the current culture. The en-us culture is the current culture on the computer on which the example was run.  
```

Comment:
Shouldn't cultures be written with the country in capitals? E.g. "en-US" here, "fr-FR" below?

Reply:
That's what I originally thought. Paging @mairaw  Acrolinx mandated all lower case.

Reply:
Although they're actually case-insensitive when supplied to the CultureInfo constructors and methods, country should be uppercase, @BIllWagner; the strings are defined by an international standard (not that I remember, of course, what the standard document is).